scripts/build-nix.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Maybe it would be nice to use this script in `scripts/test-vm.sh`?

Reply:
good idea